app/controllers/api/account.php

```diff
@@ -719,9 +719,7 @@ function sendSessionAlert(Locale $locale, Document $user, Document $project, Doc
                 continue;
             }
 
-            $dbForProject->withRequestTimestamp($requestTimestamp, function () use ($dbForProject, $session) {
-                return $dbForProject->deleteDocument('sessions', $session->getId());
-            });
+            $dbForProject->deleteDocument('sessions', $session->getId());
```

Comment:
Removed because this behavior is unused, it was added for the old offline support implementation